webrtc.html

```diff
@@ -3124,30 +3124,51 @@ <h4>
                     </p>
                     <ol>
                       <li>
-                        <p>
-                          Parse the <var>url</var> using the generic URI syntax
-                          defined in [[!RFC3986]] and obtain the <var>scheme
-                          name</var>. If the parsing based on the syntax
-                          defined in [[!RFC3986]] fails, [= exception/throw =]
-                          a {{SyntaxError}}. If the <var>scheme name</var> is
-                          not implemented by the browser [= exception/throw =]
-                          a {{NotSupportedError}}. If <var>scheme name</var> is
-                          <code class="scheme">turn</code> or <code class=
-                          "scheme">turns</code>, and parsing the <var>url</var>
-                          using the syntax defined in [[!RFC7065]] fails, [=
-                          exception/throw =] a {{SyntaxError}}. If <var>scheme
-                          name</var> is <code class="scheme">stun</code> or
-                          <code class="scheme">stuns</code>, and parsing the
-                          <var>url</var> using the syntax defined in
-                          [[!RFC7064]] fails, [= exception/throw =] a
-                          {{SyntaxError}}.
-                        </p>
+                        <p>Let <var>parsedURL</var> be the result of
+                        <a data-cite="!url#concept-url-parser">parsing</a>
+                        <var>url</var> with <code>null</code>.</p>
+                      </li>
+                      <li>
+                        <p>If <var>parsedURL</var> is failure, then
+                        [= exception/throw =] a {{SyntaxError}}.
```

Comment:
Is this a DOMException? If so, you want: `"{{SyntaxError!!exception}}" {{DOMException}}`.